content/applications/sales/sales/products_prices/loyalty_discount.rst

```diff
@@ -32,43 +32,51 @@ Or, if there are already existing programs, select an existing program to edit i
    :alt: Discount and loyalty program template cards.
 
 .. note::
-   Templates only appear when no programs have been created, and they disappear once the first
+   Templates **only** appear when no programs have been created, and they disappear once the first
    program is created.
 
 Creating or editing a program opens the program form.
 
-The program form contains the following options:
+.. image:: loyalty_discount/price-programs.png
+   :align: center
+   :alt: Program options on the loyalty program form.
 
-- :guilabel:`Program Name`: Enter the name of the program. The program name is *not* visible to the
+The program form contains the following fields:
+
+- :guilabel:`Program Name`: enter the name of the program. The program name is *not* visible to the
   customer.
-- :guilabel:`Program Type`: Select the desired :ref:`program type
+- :guilabel:`Program Type`: select the desired :ref:`program type
   <sales/pricing_management/program-types>`.
-- :guilabel:`Currency`: Select the currency used for the program.
-- :guilabel:`Points Unit`: Enter the name of the points used for the :guilabel:`Loyalty Cards`
+- :guilabel:`Currency`: select the currency used for the program.
+- :guilabel:`Pricelist`: if desired, select a pricelist from the drop-down menu to have this loyalty
+  program applied to a specific pricelist (and customers attached to the pricelist). More than one
+  pricelist can be selected in this field. When a single loyalty program is linked to several
+  pricelists, it makes it viable for different customer segments to have different pricelists, but
+  the *same* loyalty programs. If this field is left blank, the program applies to everyone,
+  regardless of pricelist.
+- :guilabel:`Points Unit`: enter the name of the points used for the :guilabel:`Loyalty Cards`
   program (e.g. `Loyalty Points`). The points unit name *is* visible to the customer. This field is
```

Comment:
I suggest making the 'only visible' part as an important admonition, or in it's own paragraph. I know you didn't edit that sentence but I can't help suggesting!

Reply:
I totally get that, and appreciate where you're coming from with the suggestion. However, just for the sake of keeping things "together" -- I think I'm going to opt to keep this wording as-is, because it quickly adds a succinct bit of information about this element, which the reader might find useful -- and I don't think it's "important enough" for its own admonition. But, as always, if someone else agrees, and thinks it should be added, I'll be sure to do so (giving you full credit for the suggestion!)